gui/guihtml/src/TGHtmlDraw.cxx

```diff
@@ -146,7 +146,7 @@ static void GetLetterIndex(char *zBuf, int index, int isUpper)
       zBuf[2] = 0;
    }
 
-   strcat(zBuf, ".");
+   strlcat(zBuf, ".", strlen(zBuf)+1);
```

Comment:
It should be something like:
```
strlcat(zBuf, ".", zBufSize);
```
And `zBufSize` should be extra argument in the function, here we do not have information
At the place where function called, you could use `sizeof(zBuf)` while it is declared as plain array `char zBuf[30]`

Reply:
Well, I think I will just add several  `// NOLINT` to disable checks. I don't want to rewrite this very old code...